### ISSUE TITLE
problem of accessibility for visually impaired people using screen reader

### ISSUE THREAD

**Issue by FrancoisLB** [thonny/thonny#3003]
Thonny version 4.1.3 for windows (portable version) is not compoliant with WCAG2.1 digital accessibility rules. 

OS : windows 11 - version 23H2

visually impaired and blind users are not able to use Thonny with
* NVDA screen reader can't annonce all menus items, buttons, fields in GUI interactions (see Non Visual Desktop Aaccess free and open source application)
*  braille terminal

in addition, a full "high contrast" color theme (fully dark, with no color) is missing in the current available list. 

many thanks by advance.

**Comment by aivarannamaa** [comment]
Please see #1178 for explanation.
